snappi_ixnetwork/device/base.py

```diff
@@ -122,6 +122,19 @@ def configure_multivalues(self, snappi_obj, ixn_obj, attr_map):
                 ))
             ixn_obj[ixn_attr] = self.multivalue(value)
 
+    def configure_multivalues_with_choice(self, snappi_obj, ixn_obj, attr_map):
```

Comment:
It will better to move `configure_multivalues_with_choice` within interface as this method only work with gateway_mac

Reply:
It might require if we get another property with choice in future